wp-content/themes/point-church/page-mission-vision-values.php

```diff
@@ -22,23 +22,15 @@
 			<div class="values">
 				<h3>Our Values</h3>
 				<ul class="styled-list">
-					<li><strong>Invest and Invite</strong> We intentionally invest our lives into people who are far from 	God and invite them to know Christ.</li>
+					<li>We are<strong>Made To Worship</strong> God created us to worship and we experience maximum joy when we worship him with all we are.</li>
```

Comment:
These 'we are's before the strong tag break the design. It expects that the strong tag will represent the 'title' for that bullet.